test/parallel/test-module-main-extension-lookup.js

```diff
@@ -4,6 +4,7 @@ const { execFileSync } = require('child_process');
 
 const node = process.argv[0];
 
-execFileSync(node, ['--experimental-modules', 'test/es-module/test-esm-ok']);
+execFileSync(node, ['--experimental-modules',
+                    'test/fixtures/es-modules/test-esm-ok']);
```

Comment:
Should this use the [`common.fixtures`](https://github.com/nodejs/node/tree/master/test/common#fixtures-module) module?

Reply:
Done, thanks